runner/runners/invoke.go

```diff
@@ -61,6 +61,7 @@ func (inv *Invoker) Run(cmd *runner.Command, id runner.RunID) (abortCh chan<- st
 func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struct{}, updateCh chan runner.RunStatus) (r runner.RunStatus) {
 	log.Infof("run. id: %v", id)
 	defer func() {
+		inv.stat.Latency(stats.WorkerTaskLatency_ms).Time().Stop()
```

Comment:
Hmm. I think the other examples work because they're the argument passed to defer, which get evaluated immediately before the defer function is executed - in this one the defer func has empty args and may not initialize the Time() until this function is already running?